### PR TITLE
CNV-19657: Clean up CNV backup/restore

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.10+

Issue: [CNV-19657](https://issues.redhat.com/browse/CNV-19657)

Link to docs preview: [Backing up and restoring virtual machines](http://file.tlv.redhat.com/apinnick/CNV-19657-clean-up-oadp/virt/backup_restore/virt-backup-restore-overview.html)

Additional information: Ready for peer review

### CODE REVIEW

**Comment by apinnick** [comment]
Ready for QE.

**Comment by aglitke** [comment]
Looks good to me.

**Comment by dalia-frank** [comment]
lgtm

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -3366,14 +3366,8 @@ Topics:
 - Name: Backup and restore
   Dir: backup_restore
   Topics:
-  - Name: Backup and restore overview
+  - Name: Backing up and restoring VMs
```
Comment (sjhala-ccs) [inline _topic_maps/_topic_map.yml:3369]: s/VMs/virtual machines

I think in our docs we have always spelled out virtual machines in the topic title

File: virt/backup_restore/virt-backup-restore-overview.adoc [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
-You restore the `Backup` CR by creating a `Restore` CR.
+.Procedure
 
+. Install the xref:../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc#about-installing-oadp[OADP Operator] according to the instructions for your storage provider.
+. Install the xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc#oadp-installing-dpa_installing-oadp-ocs[Data Protection Application] with the `kubevirt` and `openshift` xref:../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[plugins].
+. Back up virtual machines by creating a xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[`Backup` custom resource (CR)].
+. Restore the `Backup` CR by creating a xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[`Restore` CR].
+
+[role="_additional-resources"]
 [id="additional-resources_{context}"]
```
Comment (sjhala-ccs) [inline virt/backup_restore/virt-backup-restore-overview.adoc:24]: `s/{context}/virt-backup-restore-overview` per our guidelines. I think we use the variable {context} in modules.

**Review by sjhala-ccs — COMMENTED** [review]
Just a couple of nits, otherwise LGTM

**Pull request merged**